packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -289,33 +291,36 @@ servers:
     await server.stop();
   });
 
-  it('handles IPv6 loopback address in HTTPS', async () => {
-    const keyPath = path.join(__dirname, 'key.pem');
-    const certPath = path.join(__dirname, 'cert.pem');
-    const server = await givenAServer({
-      rest: {
-        port: 0,
-        host: '::1',
-        protocol: 'https',
-        key: fs.readFileSync(keyPath),
-        cert: fs.readFileSync(certPath),
-      },
-    });
-    server.handler(dummyRequestHandler);
-    await server.start();
-    const serverUrl = server.getSync(RestBindings.URL);
-    const res = await httpsGetAsync(serverUrl);
-    expect(res.statusCode).to.equal(200);
-    await server.stop();
-  });
+  process.env.TRAVIS
+    ? // tslint:disable-next-line:no-unused-expression
+      it.skip
+    : it('handles IPv6 loopback address in HTTPS', async () => {
```

Comment:
Ditto - use a helper that hides implementation complexity and keeps the code easy to read. See my earlier comment for an example.